packages/console/src/pages/Settings/components/ChangePassword.module.scss

```diff
@@ -6,7 +6,8 @@
   padding: _.unit(4);
   display: flex;
   align-items: center;
-  width: 554px;
+  justify-content: space-between;
+  @include _.form-text-field;
```

Comment:
nit: just curious, is `changePassword` a form text field?

Reply:
updated.